src/serialchemy/_tests/test_serialization.py

```diff
@@ -94,6 +87,7 @@ def test_model_load(data_regression):
         "lastname": "Kerrigan",
         "email": "sarahk@blitz.com",
         "admission": "2152-01-02T00:00:00",
+        "marital_status": "Married"
```

Comment:
I'd expect this to be the enum name. Like this. Is this correct?
```suggestion
        "marital_status": "MARRIED"
```